packages/apps/plugins/plugin-thread/src/ThreadPlugin.tsx

```diff
@@ -195,11 +208,72 @@ export const ThreadPlugin = (): PluginDefinition<ThreadPluginProvides> => {
         builder: (plugins) => {
           const client = resolvePlugin(plugins, parseClientPlugin)?.provides.client;
           const dispatch = resolvePlugin(plugins, parseIntentPlugin)?.provides.intent.dispatch;
-          if (!client || !dispatch) {
+          const metadataResolver = resolvePlugin(plugins, parseMetadataResolverPlugin)?.provides.metadata.resolver;
+
+          if (!client || !dispatch || !metadataResolver) {
             return [];
           }
 
           return [
+            createExtension({
+              id: 'comments-for-subject',
```

Comment:
generally try to prefix anything going into a global namespace with plugin id to avoid collisions